.github/dependabot.yml

```diff
@@ -24,6 +24,7 @@ updates:
       include: 'scope'
     versioning-strategy: increase
     ignore:
+      - dependency-name: "@angular/*"
       - dependency-name: "@storybook/addon-a11y"
```

Comment:
Are there any other packages that are linked to the angular upgrade? e.g. I remember `ng-mocks` was following the versioning of angular too: https://www.npmjs.com/package/ng-mocks

Reply:
I'd be weary of ignoring other packages, just incase we forgot about them when upgrading Angular. What do you think?

Reply:
We are doing the same in my project and we never forget to update whenever I update angular 🤔 
One thing I would definitely add is @angular-devkit

Reply:
Ah okay, shall we see how we go and only add `ng-mocks` if we find the Dependabot updates annoying?

I've added `@angular-devkit` 🙂